test/deprecated/writefbinary.compopts

```diff
@@ -1,2 +1,2 @@
 -st=string
--st=bytes
+#-st=bytes
```

Comment:
This comments the argument out, right?  I think you can just remove the .compopts file in that case

Reply:
I'm not sure if we still need the `.compopts` file or not, it's not clear what we are doing about binary strings.

Reply:
It looks like I also misread the changes here (how, Lydia?!), and I think it's fair to do this (though I'm not sure how we'll notice it needs to change again)